packages/webapp/src/components/Animals/AddAnimalsFormCard/AddAnimalsFormCard.tsx

```diff
@@ -111,6 +101,7 @@ export default function AddAnimalsFormCard({
         onTypeChange={(option) => {
           trigger(`${namePrefix}${BasicsFields.TYPE}`);
           onTypeChange?.(option);
+          breedSelectRef?.current?.clearValue();
```

Comment:
We possibly don't need a ref here at all, 

```js
resetField(`${namePrefix}${BasicsFields.BREED}`, { defaultValue: null });
```

seems to do the trick.
Also, not related to this PR, but do we need a ref for the UUID value? Couldn't we set it in the `useEffect` hook without storing it in a ref?

Reply:
Ah thank you!! I remember we concluded at one point that ReactSelects were impervious to hook form functions, but it was based on `setValue` not working, and I don't think I ever tried `resetField` before.

The UUID ref was meant for persisting the uuid between the component unmounting and remounting... but of course what also persists between remounts is... the form 😁🤦‍♀️ I have removed both refs.